tests/conftest.py

```diff
@@ -28,7 +28,7 @@
 
 @pytest.fixture()
 def run():
-    exp = neptune.init(mode='debug')
+    exp = neptune.init()
```

Comment:
Really this doesn't affect `test_base`?

Reply:
?

Reply:
This fixture is not only used in e2e tests, but on the other hand it's working well, so no problem.